core/domain/subtopic_page_domain.py

```diff
@@ -203,11 +251,15 @@ def get_subtopic_page_id(cls, topic_id, subtopic_id):
         return '%s-%s' % (topic_id, subtopic_id)
 
     @classmethod
-    def create_default_subtopic_page(cls, subtopic_id, topic_id):
+    def create_default_subtopic_page(
+        cls,
+        subtopic_id: int,
+        topic_id: str
+        ) -> 'SubtopicPage':
```

Comment:
```suggestion
        cls,
        subtopic_id: int,
        topic_id: str
    ) -> 'SubtopicPage':
```

can we remove the quotes?